features/onboarding/screens/OnboardingConnectWalletScreen.tsx

```diff
@@ -25,7 +25,7 @@ export const OnboardingConnectWalletScreen = memo(
 
     const handleDoneConnecting = useCallback(() => {
       if (isMissingConverseProfile()) {
-        router.navigate("OnboardingUserProfile");
+        router.navigate("OnboardingContactCard");
```

Comment:
I feel like renaming this route to `OnboardingCreateContactCard` makes more sense because that's what the route screen is doing.